### ISSUE TITLE
References not resolved

### ISSUE THREAD

**Issue by MarvelWild** [mono/cocos-sharp-samples#16]
Examples does not have references to CocosSharp.
![noref](https://cloud.githubusercontent.com/assets/7306230/16925914/226eca4a-4d2f-11e6-97b5-5eb4c0486bc4.png)
How this supposed to be resolved?

**Comment by netonjm** [comment]
You need to add the Cocosharp nuget package. This examples are outdated... this weekend I will try to fix if there is any API change

**Comment by MarvelWild** [comment]
Yes, did Install-Package CocosSharp, 
examples still not working, bit for another reason (API):
![image](https://cloud.githubusercontent.com/assets/7306230/17041597/b79a3384-4faf-11e6-8259-8ba5d65f21ed.png)

**Comment by netonjm** [comment]
Yes, this is because there was lot of CocosSharp API changes and the examples needs some code updates

**Comment by rtabbara** [comment]
@YPakhomau Please see https://github.com/mono/CocosSharp/issues/392
